src/components/TaskItem.tsx

```diff
@@ -2,9 +2,15 @@ import React from 'react';
 
 interface TaskItemProps {
   taskName: string;
+  index: number;
+  deleteTask: (value: number) => void;
 }
```

Comment:
Se seguir a sugestao acima, aqui ficaria
```suggestion
interface TaskItemProps {
  taskName: string;
  deleteTask: () => void;
}
```